README.md

```diff
@@ -10,6 +10,40 @@ This repository hosts scripts to generate self-contained installations of the [E
 
 ------
 
+
+### 1.1. Prerequisites
+
+If you don't have Git, install it. You can install Git from websites such as [Git Downloads](https://git-scm.com/downloads).
```

Comment:
Wouldn't this be cleaner if it simply read `If you don't have git, install it from [Git Downloads](https://git-scm.com/downloads).

Reply:
This seems more concise. I'll update this unless the @mcdwil or @dankhen has any objections?

Reply:
Sounds good.